docs/contributing/featureflags.md

```diff
@@ -57,63 +66,127 @@ configuration too via `FlagDataFilePath`) with:
+installations imply being offline. Set a safe default value not just in the flag definition online
+in LaunchDarkly but also in code.
+
+### Clients
+
+All clients retrieve their feature flags by querying the `/config` endpoint on the Bitwarden API.
+Clients do not directly reference the LaunchDarkly client-side SDK.
+
+In order to optimize the use of feature flags, they are not retrieved from the server on every
+request for the flag value. Rather, the flags are retrieved from the server on the following
+interval:
+
+- On application startup
+- Every hour after application startup
+- On sync (both automatic and manual)
+
+Requesting a flag value from the services defined below will provide the consuming component with
+the most recent value from one of these retrieval events.
+
+#### Web
+
+The feature flag values are retrieved through the `fetchServerConfig()` method on the
+[`ConfigService`](https://github.com/bitwarden/clients/blob/master/libs/common/src/services/config/config.service.ts).
+
+To use a feature flag, you should first define the new feature flag as an enum value in the
+[`FeatureFlags`](https://github.com/bitwarden/clients/blob/master/libs/common/src/enums/feature-flag.enum.ts)
+enum.
+
+Once that is defined, the value can be retrieved by injecting the `ConfigService` and using one of
+the retrieval methods:
+
+- `getFeatureFlagBool()`
+- `getFeatureFlagString()`
+- `getFeatureFlagNumber()`
+
+#### Mobile
+
+The feature flag values are retrieved through the `GetAsync()` method on the `ConfigService`.
+
+To use a feature flag, you should first define the new feature flag as a string constant value in
+the `Constants` file.
 
-General guidance:
+Once that is defined, the value can be retrieved by injecting the `IConfigService` and using one of
+the retrieval methods:
 
-- Title Your Feature Name.
-- Keep keys succinct.
-- Descriptions and tags aren’t needed as we’re tying flags to Jira.
-- The most likely Jira issue type for a feature link is Story.
-- Leave client-side availability unchecked altogether. We do not utilize LaunchDarkly client-side
-  and provide features states ourselves via APIs.
+- `GetFeatureFlagBoolAsync()`
+- `GetFeatureFlagStringAsync()`
+- `GetFeatureFlagNumberAsync()`
 
-Consider using flags frequently with feature development. Per the above usage of a local file,
-feature flags don’t have to exist in LaunchDarkly for initial development – **don’t create them
-online until you’re sure about the final implementation**. While there is no precise rule on what
-constitutes a "feature", work together on the best balance of flags and their respective purposes.
+### Server
+
+1. Inject `IFeatureService` where you need a feature flag. Note that you’ll also need
+   `ICurrentContext` when accessing the feature state.
+2. Check the
+   [FeatureFlagKeys](https://github.com/bitwarden/server/blob/master/src/Core/Constants.cs) constant
```

Comment:
⛏️ Your other links treat this stylistically as `code` so we should convert this too.